src/App.js

```diff
@@ -61,6 +62,7 @@ function App() {
               <Routes>
                 <Route path="/survey" element={<SurveyList />} />
                 <Route path="/survey/:surveyId" element={<EditSurvey />} />
+                <Route path="/survey/:surveyId/progress" element={<AnswerSurvey/>}/>
```

Comment:
Why is it named "progress"?

Reply:
What is wrong with progress? Users has made progress answering the survey